go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -152,11 +152,11 @@ type operationModel struct {
 	DoAfter func()
 }
 
-func onModelUpdatesNone() []interface{} {
+func OnModelUpdatesNone() []interface{} {
 	return nil
 }
 
-func onModelUpdatesAll() []interface{} {
+func OnModelUpdatesAllNonDefault() []interface{} {
```

Comment:
nonDefault is indeed a much better name.
Can we keep it private, still (onModelUpdatesAllNonDefault) ?